app/poros/forecast.rb

```diff
@@ -0,0 +1,14 @@
+class Forecast
+  attr_reader :id,
+              :timezone,
+              :current,
+              :hourly,
+              :daily
+
+  def initialize(forecast_data)
+    @timezone = forecast_data[:timezone]
+    @current = forecast_data[:current]
+    @hourly = forecast_data[:hourly]
+    @daily = forecast_data[:daily]
+  end
```

Comment:
I'm thinking I may need to add more methods later to filter out unnecessary data, but it's a start.